leo_cli_config.js

```diff
@@ -1,12 +1,6 @@
 'use strict';
 module.exports = {
 	publish: [{
-		leoaws: {
```

Comment:
I think we moved this config to an external location but if not, we need to publish to us-west-2